Sources/SpeziSecureStorage/SecureStorage.swift

```diff
@@ -306,17 +192,23 @@ public final class SecureStorage: Module, DefaultInitializable, EnvironmentAcces
     ///   - removeDuplicate: A flag indicating if any existing key for the `username` of the new credentials and `newServer`
     ///                      combination should be overwritten when storing the credentials.
     ///   - storageScope: The ``SecureStorageScope`` of the newly stored credentials.
-    public func updateCredentials( // swiftlint:disable:this function_default_parameter_at_end
+    public func updateCredential( // swiftlint:disable:this function_default_parameter_at_end
```

Comment:
We will also add a deprecated overload here; please ensure this for all methods that have changed.

Reply:
I now made sure to not change the interface of SecureStorage at all and just deprecated it as a whole.

Reply:
Sounds good 👍